### PR TITLE
Nimbledroid changes

### PR DESCRIPTION
**Pull request by armenzg**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,6 +26,7 @@
   "license": "MPL-2.0",
   "dependencies": {
     "async-redis": "^1.1.4",
+    "debug": "^4.1.1",
```
Comment (klahnakoski) [inline package.json]: Good to know about this

File: yarn.lock [hunk 2]
```diff
@@ -4975,6 +4982,11 @@ ms@2.0.0:
   resolved "https://registry.yarnpkg.com/ms/-/ms-2.0.0.tgz#5608aeadfc00be6c2901df5f9861788de0d597c8"
   integrity sha1-VgiurfwAvmwpAd9fmGF4jeDVl8g=
 
+ms@^2.1.1:
```
Comment (klahnakoski) [inline yarn.lock]: where is this used?

Comment (armenzg) [reply yarn.lock]: The changes of yarn.lock are automatic.

That's a dependency from `debug`:
https://www.npmjs.com/package/debug

File: package.json (hunk 1 above)
Comment (armenzg) [reply package.json]: 👍

**Comment by armenzg** [comment]
I tested localhost against https://firefox-health-backend-pr-56.herokuapp.com/ and it seems to work as expected.

**Pull request merged**